CONTRIBUTING.md

```diff
@@ -1,5 +1,14 @@
 # Wasabi Coding Conventions
 
+## How to be useful for the project
+
+- Any issue labelled as [good first issue](https://github.com/zkSNACKs/WalletWasabi/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22) is good to start contributing to Wasabi.
+- Always focus on a specific issue in your pull request and avoid unrelated/unnecessary changes.
+- Avoid working on complex problems (fees, amount decomposition, coin selection...) without extensive research on the context, either on Github or asking to contributors.
+- Avoid working on a UI or UX feature without first seeing a conclusion from a UX meeting such as [Extend Wallet Settings #11519](https://github.com/zkSNACKs/WalletWasabi/issues/11519).
```

Comment:
Remove that specific issue - it is confusing for new users what to see there. The sentence is enough.

Reply:
I disagree because, as a new contributor, I would have no idea what is a conclusion from a UX Meeting. Anyway, I will listen to your request.

Reply:
I agree with David that specific issue is actually confusing, because there is no actual implementation detail by the UX team, just a list of ideas.

Better examples, check last comments:
- https://github.com/zkSNACKs/WalletWasabi/issues/11128
- https://github.com/zkSNACKs/WalletWasabi/issues/10013

Reply:
Thank you for pointing this out.
I removed it, if we want later on we can add back an example.